test/bigtest/tests/forgotUsername-test.js

```diff
@@ -0,0 +1,190 @@
+import { describe, it, beforeEach } from '@bigtest/mocha';
+import { expect } from 'chai';
+import translations from '../../../translations/stripes-core/en';
+import setupApplication from '../helpers/setup-application';
+import ForgotUsernameInteractor from '../interactors/ForgotUsername';
+
+describe('Forgot username form test', () => {
+  setupApplication({ disableAuth: false });
+
+  beforeEach(function () {
+    this.visit('/forgot-username');
+  });
+
+  const forgotUsernamePage = new ForgotUsernameInteractor();
+  const {
+    inputField,
+    submitButton,
+    submitButton: { button },
+    mainHeading,
+    callToActionParagraph,
+    errorsWrapper,
+    errorsWrapper: { errorsContainer },
+    statusPage,
+    statusPage:
+        {
+          heading: statusPageHeading,
+          notificationParagraph,
+          cautionParagraph,
+        },
+  } = forgotUsernamePage;
+  const invalidInput = 'asdfgh12345';
+  const nonExistingRecord = '12345';
+  const existingRecord = '127-699-8925';
+
+  describe('Forgot form text input field tests', () => {
+    it('Should display a field to enter the email or phone number', () => {
+      expect(inputField.isPresent).to.be.true;
+    });
+
+    it('Should have an empty value', () => {
+      expect(inputField.val).to.equal('');
+    });
+  });
+
+  describe('Forgot form submit button tests', () => {
+    it('Should display a "Continue" button to submit a request', () => {
+      expect(button.isPresent).to.be.true;
+    });
+
+    it('Should have a disabled "Continue" button by default', () => {
+      expect(submitButton.isDisabled).to.be.true;
+    });
+  });
+
+  describe('Forgot form submit button test after filling the input', () => {
+    beforeEach(async () => {
+      await inputField.fillInput(invalidInput);
+    });
+    it('Should have an enabled submit button', () => {
```

Comment:
Add an empty line between `beforeEach` and `it` and between individual `it`. Be consistent with that.